docs/latest/user/deployment-mode.md

```diff
@@ -29,6 +31,12 @@ by the `marketing` and `product` teams in separate namespaces.
 helm install --set config.envoyGateway.gateway.controllerName=gateway.envoyproxy.io/marketing-gatewayclass-controller eg-marketing oci://docker.io/envoyproxy/gateway-helm --version v0.0.0-latest -n marketing --create-namespace
 ```
 
+* If you want to deploy Envoy Gateway in Namespaced mode, and watches resources only in the `marketing` namespace, you can specify a namespace by running:
```

Comment:
lets edit existing `helm` cmd instead of adding one

Reply:
updated